test/10.database.open.js

```diff
@@ -160,7 +160,7 @@ describe('new Database()', function () {
 		expect(Database.prototype).to.equal(Object.getPrototypeOf(db));
 	});
 	it('should work properly when called as a function', function () {
-		const db = this.db = Database(util.next());
+		const db = this.db = new Database(util.next());
```

Comment:
this was an issue. Maybe it could be really converted to a class and expose some factory method

Reply:
ok I see what I did wrong